alerter.c

```diff
@@ -24,8 +24,15 @@ void alertInCelcius(float farenheit) {
 }
 
 int main() {
+
+    assert(alertFailureCount == 0);

```

Comment:
Atleast 3 test values need to be covered i.e. below threshold, threshold and above threshold. If threshold is 212 F (i.e. 100 c) then consider test asserts with values <212, =212 and >212